bin/vernemq.sh

```diff
@@ -19,6 +19,22 @@ echo "########## Start ##########" >> /etc/vernemq/vernemq.conf
 
 env | grep DOCKER_VERNEMQ | grep -v DISCOVERY_NODE | cut -c 16- | tr '[:upper:]' '[:lower:]' | sed 's/__/./g' >> /etc/vernemq/vernemq.conf
 
+users_are_set=$(env | grep DOCKER_VERNE_USER)
```

Comment:
Yes, I did that to avoid more grepping in the line above. However, if you prefer, I can update `grep -v DISCOVERY_NODE` to `grep -v DISCOVERY_NODE|_USER_`?